vuepress/docs/next/docs/getting-started/README.md

```diff
@@ -139,10 +139,22 @@ Now that Kubernetes is running, you can setup kubectl to send commands directly
 
 To install Entando, we'll add `Custom Resources`, create a `Namespace`, download a `Helm` chart, and configure external access to our cluster.
 
+#### Create Namespace
```

Comment:
This reordering might be needed in the provider-specific tutorials too but I just looked at the quickstart.

Reply:
I'll check. Good catch